play-json-ops-common/src/main/scala/play/api/libs/json/ops/v4/DurationFormat.scala

```diff
@@ -132,12 +132,32 @@ object DurationFormat {
   def asString(inf: Duration.Infinite): String = inf.toString.substring("Duration.".length)
 
   object string extends StringDurationFormat {
+
+    /**
+     * In Play Json >= 2.8.0, Writes is invariant and this implicit will be used over
+     * [[ImplicitDurationWrites.durationWrites]].
+     *
+     * @note In Play Json < 2.8.0, the [[ImplicitDurationWrites.durationWrites]] is more specific than this
+     *       implicit because [[Writes]] is contravariant.
+     */
+    implicit val finiteDurationWrites: Writes[FiniteDuration] = Writes(durationWrites.writes)
```

Comment:
Play 2.8 on Scala 2.12 shares this same code base, so I have to shim this in. Thankfully, it is a minor change that does not affect Play 2.7 versions, because contravariance rules favor more generic implicits over more specific types, so this only gets picked up when `Writes` becomes invariant in Play 2.8.